test/mock_praw.py

```diff
@@ -0,0 +1,36 @@
+import time
+
+class Redditor():
+    def __init__(self, name):
+        self.name = name
+
+    def __str__(self):
+        return self.name
+
+class Submission(str):
+    def __new__(cls, id, *args):
```

Comment:
l would recommend to change the `id` variable to something else as in python, `id` is a reserved keyword. It's really weird why they did that, idk. You can leave `id` is a var name in a method, so `self.id` or `comment.id` are okay to use but just using `id` alone (as a var) might be troublesome. You can rename it to `ID` or some C++ style `id_m`. Thanks